lib/android/src/main/java/com/airbnb/android/react/maps/MapsPackage.java

```diff
@@ -24,7 +24,7 @@ public MapsPackage() {
     return Arrays.<NativeModule>asList(new AirMapModule(reactContext));
   }
 
-  @Override
+  // Deprecated RN 0.47
```

Comment:
I would remove the `@Override` part instead of the comment. Since comments get old very fast